plugins/atom-cygwin/atom-cygwin.plugin.zsh

```diff
@@ -0,0 +1,18 @@
+if [[ "$OSTYPE" == "cygwin" ]]; then
+    local _atom_path > /dev/null 2>&1
```

Comment:
You're welcome.

I forgot to mention: as a general rule of thumb, whenever you end up redirecting the output of `local`, `typeset`, or something similar to `>/dev/null`, that's a sign it's being used incorrectly.